configs/config.toml.SAMPLE

```diff
@@ -37,6 +37,12 @@
 	# including view function calls.
 	#
 	# ConcurrencyLimit = 30
+	#
+	# BalanceAlertThreshold defines a minimum value of the operator's account
+	# balance below which the client will start reporting errors in logs.
+	# The value should be passed in [wei].
+	#
+	# BalanceAlertThreshold = 500000000000000000 # 0.5 ETH (default value)
```

Comment:
We should probably accept this value in Gwei. Right now, I can not set up a threshold alert for anything higher than 10 ETH. For small operators it is enough but for big ECDSA whales it's not enough if gas prices will skyrocket. 

Also, we accept `MaxGasPrice` in `Gwei` so those two values would be aligned.

Reply:
`MaxGasPrice` is passed in `wei` as well

Reply:
I see. OK, let's leave it as-is for now. Do you mind opening a separate issue to change `MaxGasPrice` and `BalanceAlertThreshold` to Gwei?

Reply:
Here it is: https://github.com/keep-network/keep-core/issues/2176